Jenkinsfile

```diff
@@ -692,14 +698,18 @@ def runConjurTests(run_only_str) {
     ]
   ]
 
-  // Filter for the tests we want run, if requested.
+  // Filter for thez tests we want run, if requested.
   parallel_tests = all_tests
   tests = run_only_str.split()
-
+  black_list = not_run_str.split()
   if (tests.size() > 0) {
     parallel_tests = all_tests.subMap(tests)
   }
-
+  if (black_list.size() > 0){
+    black_list.each { key ->
+      parallel_tests.remove(key)
+    }
+  }
```

Comment:
Is it possible to handle the logic whether we run a test only in the function `testShouldRun()`?
Didn't fully think this through but maybe something like this could work:
`(run_only_str == '' && not_run_str =='') || (run_only_str.split().contains(test) && !not_run_str.split().contains(test))`